soh/soh/Enhancements/audio/AudioCollection.cpp

```diff
@@ -160,23 +161,162 @@ AudioCollection::AudioCollection() {
         SEQUENCE_MAP_ENTRY(NA_SE_EN_FANTOM_ST_LAUGH,     "Phantom Ganon Laugh",                 "NA_SE_EN_FANTOM_ST_LAUGH",       SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_EV_PLANT_BROKEN,        "Plant Explode",                       "NA_SE_EV_PLANT_BROKEN",          SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_SY_GET_RUPY,            "Rupee",                               "NA_SE_SY_GET_RUPY",              SEQ_SFX),
-        SEQUENCE_MAP_ENTRY(NA_SE_VO_RT_CRASH,            "Ruto Crash",                          "NA_SE_VO_RT_CRASH",              SEQ_SFX),
-        SEQUENCE_MAP_ENTRY(NA_SE_VO_RT_LIFT,             "Ruto Lift",                           "NA_SE_VO_RT_LIFT",               SEQ_SFX),
-        SEQUENCE_MAP_ENTRY(NA_SE_VO_RT_THROW,            "Ruto Thrown",                         "NA_SE_VO_RT_THROW",              SEQ_SFX),
+        //SEQUENCE_MAP_ENTRY(NA_SE_VO_RT_CRASH,            "Ruto Crash",                          "NA_SE_VO_RT_CRASH",              SEQ_SFX),
+        //SEQUENCE_MAP_ENTRY(NA_SE_VO_RT_LIFT,             "Ruto Lift",                           "NA_SE_VO_RT_LIFT",               SEQ_SFX),
+        //SEQUENCE_MAP_ENTRY(NA_SE_VO_RT_THROW,            "Ruto Thrown",                         "NA_SE_VO_RT_THROW",              SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_EN_NUTS_UP,             "Scrub Emerge",                        "NA_SE_EN_NUTS_UP",               SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_EN_AWA_BOUND,           "Shabom Bounce",                       "NA_SE_EN_AWA_BOUND",             SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_EN_SHELL_MOUTH,         "Shellblade",                          "NA_SE_EN_SHELL_MOUTH",           SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_EN_STALTU_DAMAGE,       "Skulltula Damage",                    "NA_SE_EN_STALTU_DAMAGE",         SEQ_SFX),
         SEQUENCE_MAP_ENTRY(NA_SE_EN_NUTS_THROW,          "Spit Nut",                            "NA_SE_EN_NUTS_THROW",            SEQ_SFX),
-        SEQUENCE_MAP_ENTRY(NA_SE_VO_TA_CRY_0,            "Talon Hmm",                           "NA_SE_VO_TA_CRY_0",              SEQ_SFX),
-        SEQUENCE_MAP_ENTRY(NA_SE_VO_TA_SURPRISE,         "Talon Surprised",                     "NA_SE_VO_TA_SURPRISE",           SEQ_SFX),
```

Comment:
I assume the entries here in the top section were commented out because they were re-added below. Could you instead just remove the commented out entries?

Reply:
Will do